app/controllers/api/account.php

```diff
@@ -530,6 +532,11 @@
 
         Authorization::setRole('user:'.$user->getId());
 
+        $session = $dbForInternal->createDocument('sessions', $session
+            ->setAttribute('$read', ['user:'.$user->getId()])
+            ->setAttribute('$write', ['user:'.$user->getId()])
+        );
+
         $user = $dbForInternal->updateDocument('users', $user->getId(), $user);
```

Comment:
The new session is not merged with the user yet right?

Reply:
We first insert the session to its own collection and then merge it to the user object on line 540

Reply:
Perfect 👍🏻